docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.7"
+services:
+  elasticmq-native:
+    container_name: elasticmq-container
```

Comment:
Name clashed with a container from elsewhere. Do we need to name it?

Reply:
Renamed.

Reply:
Confirmed.